AnnoDesigner/MainWindow.xaml

```diff
@@ -93,6 +93,33 @@
                     <MenuItem Header="{l:Localize LoadLayoutFromJson}"
                               Command="{Binding LoadLayoutFromJsonCommand}" />
                     <Separator />
+                    <MenuItem Header="{l:Localize RecentFiles}"
+                              ItemsSource="{Binding RecentFiles}">
+                        <MenuItem.Style>
```

Comment:
Replace entire style with MenuItem's property `IsEnabled={Binding HasRecentFiles}`?

Reply:
That would be too easy 😁
I'll do it anyway.